watsononlinestore/watson_online_store.py

```diff
@@ -83,6 +85,46 @@ def handle_db_lookup(self):
             print("DB.\n context:{}\n".format(self.context))
         return False
 
+    def handle_lookupAndAddEmail(self):
+        """ Verify email is not in DB and add to DB.
+        """
+        email_addr = str(self.context['email'])
+        if "mailto" in email_addr:
+            email_addr = self.cleanup_email(email_addr)
+        if DEBUG:
+            print("DB.\n email_addr:{}\n".format(email_addr))
+        existing_user = self.cloudant_online_store.find_customer(email_addr)
+        if DEBUG:
+            print("DB.\n existing_user:{}\n".format(existing_user))
+        if existing_user:
+            #return some error
+            return False
+        
+        # start to create the customer, add to DB when complete
+        self.customer = OnlineStoreCustomer(email=email_addr)
+
+        return True
+
+    def handle_AddName(self):
+        """ Add User Name to existing OnlineStoreCustomer
+        """
+        full_name = str(self.context['full_name'])
+        first, last = full_name.split()
```

Comment:
you could do .split(" ", 1) to only perform 1 split at most, on the first space. it may not split things properly, but it'll make sure we don't get up with trying to set 3 or more items to a tuple